authentication/authentication.md

```diff
@@ -0,0 +1,12 @@
+# Authentication
+
+## Description
+The authentication required for the Shakti API endpoint generally consists of two token values: NetflixId and SecureNetflixId. These are both passed in as cookie values for requests with every endpoint, unless otherwise stated.
+
+Obviously, this means that in order to hit the API endpoints, you must be subscribed to Netflix!
```

Comment:
> "Obviously, ..." 

Change this to a more *passive* tone. Perhaps swap obviously to "Therefore, ..." or simply remove it altogether.